docs/tutorial.rst

```diff
@@ -853,6 +853,27 @@ Both the schema and the metadata can be retrieved from a :class:`tskit.TreeSeque
 
     'Bob1234'
 
+Top-level metadata can be set on a :class:`tskit.TableCollection` and read from a
+:class:`tskit.TreeSequence`:
+
+.. code-block:: python
+
+    tables = tskit.TableCollection(sequence_length=1)
+    tables.metadata_schema = tskit.MetadataSchema(
+        {
+            "codec": "json",
+            "properties": {"author": {"type": "string"}, "version": {"type": "number"}},
```

Comment:
I'm not blown away by this example - it's not really showing the crucial stuff that we want people to associate with a tree sequence so that people can understand what it means. Examples are important, and people have a very strong habit of copying them and assuming that they represent best practise. 

But, maybe we should just put this off until we do the proper treatment of standardised tree sequence metadata keys (which we don't want to get bogged down in now). I'd say just delete the example.

Reply:
Yeah, should have thought more about that one. Deleted for now.